plutus-core/plutus-ir/test/TransformSpec.hs

```diff
@@ -60,7 +63,15 @@ nonStrict = testNested "nonStrict"
 letFloat :: TestNested
 letFloat =
     testNested "letFloat"
-    $ map (goldenPir (LetMerge.letMerge . RecSplit.recSplit . LetFloat.floatTerm . runQuote . PLC.rename) $ term @PLC.DefaultUni @PLC.DefaultFun)
+    $ map (goldenPirM
+              (\ pir -> rethrow . asIfThrown @(PIR.Error PLC.DefaultUni PLC.DefaultFun ()) $ do
```

Comment:
You know what I'm going to say... another multi-line parenthesized expression. Just pull it out!